agent/p/agent/tasks/continuous/task.go

```diff
@@ -0,0 +1,124 @@
+package continuous
+
+import (
+	"bufio"
+	"bytes"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gnolang/gno/agent/p/agent/task"
+	"gno.land/p/demo/avl"
+)
+
+type Task struct {
+	id                  string
+	Era                 int
+	NextDue             time.Time
+	Interval            time.Duration
+	Aggregator          task.Aggregator
+	definition          task.Definition
+	RespondentWhiteList *avl.Tree
```

Comment:
What about switching to slices?

Reply:
For performance reasons under the assumption that whitelists will never be very large or to make them simpler to work with without the code using it needing the knowledge of how to interact with an avl tree?

`Tasks` have been rebranded to `Feeds` in the new version and no longer contain a whitelist.  This has been moved outside of the `Feed` type to keep the interface a bit smaller.  It is now an interface with methods that do a subset of what avl.Tree does and allows for adding multiple addresses with one method call. Anyone can implement their own `Whitelist` type if they don't want to use the default -- multiple levels of permission, conditional checks, etc.